src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureQC.h

```diff
@@ -86,17 +86,17 @@ namespace OpenMS
 
       // Feature members
       /// retention time lower bound
-      double retention_time_l;
+      double retention_time_l { 0.0 };
       /// retention time upper bound
-      double retention_time_u;
+      double retention_time_u { 100.0 };
```

Comment:
Let's change this to 1e12 with the above in mind.  This is different than my previous recommendation, but probably better than my previous as well.